concent_api/core/tests/test_utils.py

```diff
@@ -120,3 +125,40 @@ def test_that_both_subtask_verification_time_implementation_should_return_same_r
                 calculate_subtask_verification_time(report_computed_task),
                 int(helpers.subtask_verification_time(report_computed_task).total_seconds())
             )
+
+
+class StoreOrUpdateSubtaskTest(ConcentIntegrationTestCase):
```

Comment:
Why doing this using subclass of `ConcentIntegrationTestCase` ? Couldn't this be done using method defined in `TestEnsureRetryOfLockedCalls` ?